.travis.yml

```diff
@@ -1,18 +1,17 @@
 sudo: false
 language: node_js
 cache:
-  directories:
-    - node_modules
+  yarn: true
 notifications:
   email: false
 node_js:
   - '6'
 before_install:
-  - npm i -g npm@^3.0.0
-before_script:
-  - npm prune
+  - npm i -g yarn
```

Comment:
I think if you include yarn as the cache strategy that yarn will get installed by travis.